tools/serverpod_cli/test/generator/dart/server_code_generator/future_calls_test.dart

```diff
@@ -0,0 +1,323 @@
+import 'package:path/path.dart' as path;
+import 'package:recase/recase.dart';
+import 'package:serverpod_cli/src/analyzer/protocol_definition.dart';
+import 'package:serverpod_cli/src/generator/dart/server_code_generator.dart';
+import 'package:test/test.dart';
+
+import '../../../test_util/builders/future_call_definition_builder.dart';
+import '../../../test_util/builders/future_call_method_definition_builder.dart';
+import '../../../test_util/builders/generator_config_builder.dart';
+import '../../../test_util/builders/parameter_definition_builder.dart';
+import '../../../test_util/builders/type_definition_builder.dart';
+
+const projectName = 'example_project';
+final config = GeneratorConfigBuilder().withName(projectName).build();
+const generator = DartServerCodeGenerator();
+
+void main() {
+  var expectedFutureCallsFileName = path.join(
+    'lib',
+    'src',
+    'generated',
+    'future_calls.dart',
+  );
+
+  var expectedEndpointsFileName = path.join(
+    'lib',
+    'src',
+    'generated',
+    'endpoints.dart',
+  );
+
+  group(
+    'Given protocol definition without future calls when generating future calls file',
```

Comment:
Suggestion: Since we have a group about the endpoints file and another about the future calls file, it will be clearer to remove the "file" from the given. Same for all groups in this file.

```suggestion
    'Given protocol definition without future calls when generating future calls',
```